src/background/hypothesis-chrome-extension.js

```diff
@@ -120,14 +120,28 @@ export default function HypothesisChromeExtension({
     });
   }
 
+  /**
+   *
+   * @param {number} tabId
+   * @param {import('./tab-state').State|undefined} current
+   */
   function onTabStateChange(tabId, current) {
     if (current) {
-      browserAction.update(tabId, current);
```

Comment:
This is moved inside the callback below (`chromeTabs.get`) and only executed if there is no errors getting the tab.